jar-infer/src/main/java/com/uber/nullaway/jarinfer/definitelyDerefedParamsDriver.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2018. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.nullaway.jarinfer;
+
+import com.ibm.wala.cfg.ControlFlowGraph;
+import com.ibm.wala.classLoader.IMethod;
+import com.ibm.wala.ipa.callgraph.AnalysisCache;
+import com.ibm.wala.ipa.callgraph.AnalysisCacheImpl;
+import com.ibm.wala.ipa.callgraph.AnalysisOptions;
+import com.ibm.wala.ipa.callgraph.AnalysisScope;
+import com.ibm.wala.ipa.callgraph.impl.Everywhere;
+import com.ibm.wala.ipa.cha.ClassHierarchyException;
+import com.ibm.wala.ipa.cha.ClassHierarchyFactory;
+import com.ibm.wala.ipa.cha.IClassHierarchy;
+import com.ibm.wala.ssa.IR;
+import com.ibm.wala.ssa.ISSABasicBlock;
+import com.ibm.wala.ssa.SSAInstruction;
+import com.ibm.wala.types.ClassLoaderReference;
+import com.ibm.wala.types.MethodReference;
+import com.ibm.wala.util.config.AnalysisScopeReader;
+import com.ibm.wala.util.io.CommandLine;
+import com.ibm.wala.util.strings.Atom;
+import com.ibm.wala.util.strings.ImmutableByteArray;
+import com.ibm.wala.util.strings.UTF8Convert;
+import com.ibm.wala.util.warnings.Warnings;
+import java.io.IOException;
+import java.util.*;
+import com.google.common.base.Preconditions;
+
+/*
+ * Driver for running {@link DefinitelyDerefedParams}
+ */
+public class DefinitelyDerefedParamsDriver {
+  private static final String PRIMORDIAL_SCOPE_FILE_PATH = "./src/test/resources/com/uber/nullaway/jarinfer/testdata/test.scope.txt";
```

Comment:
Not blocking, but: Wonder if you can read this from the Jar resources, rather than hardcode the path ( see e.g. https://alvinalexander.com/blog/post/java/read-text-file-from-jar-file )